docs/conf/sponsorship/prospectus.rst

```diff
@@ -0,0 +1,214 @@
+
+For too long, people who care about documentation have felt alone in the
+world, not able to connect with their community. Write the Docs is a
+magical experience for many of our attendees, allowing them to feel like
+they have found their place. We believe this is the most
+important thing that our event can do, and we aim to provide that
+experience again this year, and make it better than ever.
+
+Demographics
+=============
+
+We hold two conferences yearly, one in Portland, USA in May, and one in
+Europe, currently Prague, in September. Attendence is approximately 400 in the
+USA and 300 in Europe, and we expect both conferences to sell out.
+
+Our audience is made up of:
+
+-  Technical Writers (40%)
+
+-  Developers (22%)
+
+-  Support Staff (15%)
+
+-  Managers (12%)
+
+-  Community Contributors, Enthusiasts & Other Folks (11%)
+
+Why Sponsor
+=============
+
+By supporting a Write the Docs event, your company will gain not only visibility
+and credibility with front-line technical communicators, but also valuable
+insights that will help you get the most out of your own documentation efforts.
+If you're hiring for docs positions, Write the Docs is also an excellent
+opportunity to meet top-notch talent.
+
+Our conferences are run by team of volunteers, and we work hard to keep ticket
+prices affordable for a broad range of attendees. Your sponsorship makes it
+possible for all sorts of documentarians to attend our events, whether they're a
+solo freelancer or they run a docs team for a tech giant. Becoming a sponsor
+demonstrates your commitment to and support of good documentation, and the
+people who build it.
+
+.. raw:: pdf
+
+   PageBreak
+
+Sponsorship Packages
+====================
+
+The following options are suggested sponsorship levels. We are happy to discuss
+adjustments and custom packages.
+
+Each of these packages applies to *one* conference, either USA or Europe. Ask us
+about discounts packages for sponsoring both conferences.
+
+First Draft
+--------------------
+
+The **First Draft** package is aimed at startups, non-profits and open source
+companies:
+
+-  One (1) tickets
+
+-  Small logo & link on the Write the Docs website
+
+-  Name included in press release and event mailings
+
+-  Display 1 promotional (“Swag”) item at the conference (provided by
+       sponsor)
+
+The **First Draft** package costs **US$750**
+
+
+Second Draft
+----------------
+
+The **Second Draft** package for when you are no longer a startup:
+
+-  Two (2) tickets
+
+-  Large logo & link on the Write the Docs website
+
+-  Name included in press release and event mailings
+
+-  Display 1 promotional (“Swag”) items at the conference (provided by
+       sponsor)
+
+The **Second Draft** package costs **US $1500**
+
+Publisher
+---------------------------
+
+The **Publisher** package highlights your company as a force in the industry and
+community:
+
+-  Five (5) tickets
+
+-  Full size logo & link on the Write the Docs website
```

Comment:
ah, I would change "full size" to "large" and make the 2nd level "medium", probably easier to compare this way